homeassistant/components/automation/__init__.py

```diff
@@ -374,6 +379,7 @@ def _check_size_limit(self):
 def trace_automation(hass, unique_id, config, trigger, context):
     """Trace action execution of automation with automation_id."""
     automation_trace = AutomationTrace(unique_id, config, trigger, context)
+    trace_id_set((unique_id, automation_trace.runid))
```

Comment:
We needed to rebase the PR. `runid` attribute has been renamed to `run_id` on dev branch.

Reply:
Fixed by #47705